content/webapp/services/prismic/transformers/events.ts

```diff
@@ -89,10 +99,12 @@ export function transformEvent(
       }
     : undefined;
 
-  const thirdPartyBooking = data.thirdPartyBookingName
+  const thirdPartyBooking: ThirdPartyBooking | undefined = isDocumentLink(
+    data.thirdPartyBookingUrl
+  )
     ? {
-        name: data.thirdPartyBookingName,
-        url: data.thirdPartyBookingUrl,
+        name: data.thirdPartyBookingName || undefined,
+        url: data.thirdPartyBookingUrl.url!,
```

Comment:
Another thing caught by TypeScript, in the camp of "how did this ever work at all".